pkg/envoy/rds/builder.go

```diff
@@ -0,0 +1,210 @@
+package rds
+
+import (
+	"github.com/envoyproxy/go-control-plane/pkg/cache/types"
+	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	xds_route "github.com/envoyproxy/go-control-plane/envoy/config/route/v3"
+
+	"github.com/openservicemesh/osm/pkg/envoy"
+	"github.com/openservicemesh/osm/pkg/envoy/rds/route"
+	"github.com/openservicemesh/osm/pkg/trafficpolicy"
+)
+
+const (
+	// inboundVirtualHost is prefix for the virtual host's name in the inbound route configuration
+	inboundVirtualHost = "inbound_virtual-host"
+
+	// outboundVirtualHost is the prefix for the virtual host's name in the outbound route configuration
+	outboundVirtualHost = "outbound_virtual-host"
+
+	// egressVirtualHost is the prefix for the virtual host's name in the egress route configuration
+	egressVirtualHost = "egress_virtual-host"
+
+	// ingressVirtualHost is the prefix for the virtual host's name in the ingress route configuration
+	ingressVirtualHost = "ingress_virtual-host"
+)
+
+type routesBuilder struct {
```

Comment:
i think the builder and the route package should be joined together. It would make sense to me to move this file into pkg/envoy/rds/route

Reply:
As per Shashank's comment I will be removing `/route` and moving everything to `/rds`